src/Badge.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Badge extends React.Component {
+  render() {
+    let classes = ['badge'];
+    if (this.props.text) {
+      classes.push(this.props.text);
```

Comment:
Yeah, sorry, I was running on very little sleep,  'new' should definitely be a boolean prop not a string.  I can close the PR , make the changes and submit a new one.